core/src/main/java/org/testcontainers/dockerclient/DockerClientProviderStrategy.java

```diff
@@ -93,7 +97,10 @@ public static DockerClientProviderStrategy getFirstValidStrategy(List<DockerClie
                                         LOGGER.warn("Can't instantiate a strategy from {}", it, e);
                                         return Stream.empty();
                                     }
-                                }),
+                                })
```

Comment:
is the formatting in this Stream.concat... block just 'idea' being weird?  doesnt seem to follow the 4 space indents everywhere else.

Reply:
This class is not modified for this PR, probably upstream has changed and so you see the difference.

Reply:
Please let's see if we can tackle this without regular expressions separately. I'd be very happy if we could do that and make this code easier to read.